platform/web/js/engine/features.js

```diff
@@ -84,11 +84,16 @@ const Features = { // eslint-disable-line no-unused-vars
 		if (!Features.isSecureContext()) {
 			missing.push('Secure Context - Check web server configuration (use HTTPS)');
 		}
-		if (!Features.isCrossOriginIsolated()) {
-			missing.push('Cross Origin Isolation - Check web server configuration (send correct headers)');
-		}
-		if (!Features.isSharedArrayBufferAvailable()) {
-			missing.push('SharedArrayBuffer - Check web server configuration (send correct headers)');
+
+		// eslint-disable-next-line no-undef
+		const compiledWithThreads = ___GODOT_THREADS_ENABLED;
```

Comment:
We should have `getMissingFeatures` take a parameter, and do the substitution in the html file.
```
	getMissingFeatures: function (wantsThreads = true) {
		// ...
		if (wantsThreads) {
			// ...
		}
		// ...
	},
```